src/main/java/DateTime.java

```diff
@@ -0,0 +1,121 @@
+public class DateTime {
+
+    String dateTime;
+    String day;
+    String month;
+    String year;
+    String time;
+
+    public DateTime(String dateTime) {
+        this.dateTime = dateTime;
+    }
+
+    public void processDate(String d) {
```

Comment:
Perhaps the 'd' character in the params of this can be replaced with something more meaningful according to the coding standards convention

Reply:
true esp for team project